manifests/profiles/version.yaml

```diff
@@ -5,7 +5,7 @@ appVersion: "1.12.0"
 
 cloudCore:
   replicaCount: 1
-  hostNetWork: "true"
+  hostNetWork: false
   image:
     repository: "kubeedge/cloudcore"
     tag: "v1.12.0"
```

Comment:
Line#55 in this file should be "  enable: "true"" ?

Reply:
yes